addons/point_of_sale/models/pos_bill.py

```diff
@@ -19,3 +20,9 @@ def name_create(self, name):
             raise UserError(_("The name of the Coins/Bills must be a number."))
         result = super().create({"name": name, "value": value})
         return result.id, result.display_name
+
+    @api.depends("for_all_config")
+    def _compute_pos_config_ids(self):
+        for record in self:
+            if record.for_all_config:
+                record.write({"pos_config_ids": [Command.clear()]})
```

Comment:
Clearing this when changing `for_all_config` will make the missclick quite annoying when you could just keep it like this and, if `for_all_config` is `True`, not look at this field. What do you think ?

Reply:
You're right. I'll make the change.

Reply:
Review comment resolved a separate commit.